### DISCUSSION TITLE
Hand sorting

### DISCUSSION THREAD

**Discussion by kostya-phoenix** [bafolts/terraforming-mars#2929] [Ideas]
Hi!
I think it's quite handy :) to be able to sort your hand by cost and by requirements.
At least I'm doing so while playing a real game.

Also, I have an idea to display cards in one row which you can easily scroll by mouse wheel.
Current drag & drop is not satisfying, with 1 line it should be easier.

Any feedback are welcome.

**Comment by bafolts** [comment]
Displaying in one row would line up with a goal of trying to get everything to display on one page without having to scroll the page. The hand sorting probably warrants a separate discussion from displaying the cards in a row.

**Comment by Arthurin** [comment]
I like both ideas !
